StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentOptionsViewController.swift

```diff
@@ -682,7 +682,7 @@ extension STPPaymentMethod {
     var removalMessage: (title: String, message: String) {
         switch type {
         case .card:
-            let brandString = STPCardBrandUtilities.stringFrom(card?.networks?.preferred?.toCardBrand ?? card?.brand ?? .unknown) ?? ""
+            let brandString = STPCardBrandUtilities.stringFrom(card?.preferredDisplayBrand ?? .unknown) ?? ""
```

Comment:
Do we know when we should be using `preferredDisplayBrand` vs card?.networks?.preferred?  The description says that we should "fallback" but this approach seems to use the preferredDisplayBrand and fallback to unknown.

Reply:
preferredDisplayBrand is

```swift
    var preferredDisplayBrand: STPCardBrand {
        return networks?.preferred?.toCardBrand ?? displayBrand?.toCardBrand ?? brand
    }
```

Reply:
So we attempt to read the preferred brand, if nil fallback to displayBrand then fallback to brand.